src/samplers/aliastable.jl

```diff
@@ -1,21 +1,20 @@
-struct AliasTable{S} <: Sampleable{Univariate,Discrete}
+struct AliasTable{N} <: Sampleable{Univariate,Discrete}
     accept::Vector{Float64}
     alias::Vector{Int}
-    isampler::S
 end
-ncategories(s::AliasTable) = length(s.accept)
+ncategories(s::AliasTable{N}) where N = N
 
 function AliasTable(probs::AbstractVector{T}) where T<:Real
```

Comment:
Since you're at it - it seems `T` is not used anywhere in the function body, so maybe remove it?

Reply:
Good catch. Done